heron/tools/tracker/src/python/routers/container.py

```diff
@@ -143,8 +142,8 @@ class ExceptionLog(BaseModel):
   hostname: str
   instance_id: str
   stack_trace: str = Field(..., alias="stacktrace")
-  last_time: int = Field(..., alias="lasttime")
-  first_time: int = Field(..., alias="firsttime")
+  last_time: str = Field(..., alias="lasttime")
+  first_time: str = Field(..., alias="firsttime")
```

Comment:
We need to be careful with this. One of the many oddities and quirks of Python is that assigning values of different types to a variable will convert the type of the variable.

```python
test: str = "test string"
print (type(test))

test = 6
print (type(test))

#####

number: int = -1
print(type(number))

test = "test string"
number = test
print(type(number))
```